### PR TITLE
fix: social media - only part 1

### PR DESCRIPTION
**Pull request by kotzendekrabbe**

### CODE REVIEW

File: data/jobs/de/data-engineer.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
     date: 2020-10-29
     published: true
     title: Data Engineer
-    
+    description: hier
```
Comment (kotzendekrabbe) [inline data/jobs/de/data-engineer.md:10]: for testing

File: template/partials/head.html [hunk 2]
```diff
@@ -7,6 +7,21 @@
     <meta name="language" content="{{lang}}">
     <meta name="viewport" content="width=device-width">
     <meta name="robots" content="index,follow">
+    
+    <meta property="fb:page_id" content="358531227139">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="SinnerSchrader Career">
```
Comment (chaosmonster) [inline template/partials/head.html:13]: maybe use `pageTitle`

File: template/partials/head.html [hunk 3]
```diff
@@ -7,6 +7,21 @@
...
+    <meta property="og:description" content="We believe two heads are always better than one. And a whole team of heads is even better. That’s how we work. With no alpha leaders and no beta teams.">
```
Comment (chaosmonster) [inline template/partials/head.html:14]: maybe use `pageDesc`

File: template/partials/head.html [hunk 4]
```diff
@@ -7,6 +7,21 @@
...
+    <meta property="og:url" content="https://sinnerschrader.jobs">
+    <meta property="og:image" content="./assets/images/sinnerschrader.png">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:url" content="https://sinnerschrader.jobs">
+    <meta name="twitter:site" content="@sinnerschrader">
+    <meta name="twitter:creator" content="@sinnerschrader">
+    <meta name="twitter:title" content="SinnerSchrader Career">
```
Comment (chaosmonster) [inline template/partials/head.html:21]: maybe use `pageTitle`

File: template/partials/head.html [hunk 5]
```diff
@@ -7,6 +7,21 @@
...
+    <meta name="twitter:description" content="We believe two heads are always better than one. And a whole team of heads is even better. That’s how we work. With no alpha leaders and no beta teams.">
```
Comment (chaosmonster) [inline template/partials/head.html:22]: maybe use `pageDesc`

File: template/de/index.html [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="de">
-{{>head baseHref=baseHref pageTitle="SinnerSchrader Jobs - de Title" pageDesc="some description" lang="de"}}
+{{>head baseHref=baseHref pageTitle="SinnerSchrader Career" pageDesc="We believe two heads are always better than one. And a whole team of heads is even better. That’s how we work. With no alpha leaders and no beta teams." lang="de"}}
```
Comment (chaosmonster) [inline template/de/index.html:3]: Do we want to use the English description?

File: template/partials/head.html (hunk 2 above)
Comment (kotzendekrabbe) [reply template/partials/head.html:13]: how?

File: template/partials/head.html (hunk 3 above)
Comment (kotzendekrabbe) [reply template/partials/head.html:14]: see above - how?

File: template/de/index.html (hunk 6 above)
Comment (kotzendekrabbe) [reply template/de/index.html:3]: at the moment? yes

**Review by chaosmonster — APPROVED** [review]

**Pull request merged**